src/openms/include/OpenMS/CONCEPT/LogStream.h

```diff
@@ -469,7 +469,12 @@ namespace OpenMS
   /// Macro for general debugging information
 #define OPENMS_LOG_DEBUG \
   OPENMS_THREAD_CRITICAL(LOGSTREAM) \
-  OpenMS_Log_debug << OPENMS_FILENAME << "(" << __LINE__ << "): "
+  OpenMS_Log_debug << [](){ constexpr const char * x = (past_last_slash(__FILE__)); return x; }() << "(" << __LINE__ << "): "
+
+  /// Macro for general debugging information (without information on file)
+#define OPENMS_LOG_DEBUG_NOFILE \
```

Comment:
Having two macros seems adding complexity. Do we need the NOFILE macro? But maybe not that urgent right now.

Reply:
I think so. There are a lot of places where people think they can just stack Debug statements. If you see that there are two macros, you think twice about which to pick.